app/endpoints/users.py

```diff
@@ -156,17 +156,17 @@ async def create_user_by_user(
     # Check the account type
 
     # For staff and student
-    # ^[\w\-.]*@(ecl\d{2})|(alternance\d{4})|(auditeur)|(master)?.ec-lyon.fr$
+    # ^[\w\-.]*@((etu(-enise)?|auditeur).)?ec-lyon.fr$
     # For staff
     # ^[\w\-.]*@(auditeur.)?ec-lyon.fr$
     # For student
-    # ^[\w\-.]*@((ecl\d{2})|(alternance\d{4})|(master)).ec-lyon.fr$
+    # ^[\w\-.]*@etu(-enise)?.ec-lyon.fr$
 
-    if re.match(r"^[\w\-.]*@(auditeur.)?ec-lyon.fr", user_create.email):
+    if re.match(r"^[\w\-.]*@(auditeur.)?ec-lyon.fr$", user_create.email):
```

Comment:
auditeur.ec-lyon.fr email address have been replaced by @etu.ec-lyon.fr
enise.ec-lyon.fr exists for staff at ENISE

Reply:
This is something we did not know. This will require a change in Titan too @maximeroucher

Reply:
Got it !